config/vufind/config.ini

```diff
@@ -596,6 +596,15 @@ username_hint = username_only_letters_numbers_and_basic_punctuation
 ; Default is true.
 ;delete_ratings_with_user = false
 
+; "Remember me" functionality for listed login methods. If the user chooses to save the login,
+; a login token cookie and a database entry will be created. Users browser information will be 
```

Comment:
Should add an apostrophe to "Users".

Reply:
Fixed. Also removed the last sentence "Login methods are written in lowercase." because it is not needed.